cogs/rcon.py

```diff
@@ -0,0 +1,262 @@
+        servers = ["eu", "us", "eumvm", "usmvm"]
+
+        #only accept valid servers
+        if server in servers:
+
+            #rcon to specific server
+            #eu
+            if server == "eu":
+                command = " ".join(commands)
+                response = await run_rcon(server, command)
+
+                #for status
+                if command == "status":
+                    status_response = await get_status(response)
+                    await ctx.send(status_response)
+                else:
+                #for everything else
+                    if response == "":
+                        await ctx.send("Command sent.")
+                    else:
+                        await ctx.send("```\n" + response + "\n```")
+            #us
+            elif server == "us":
+                command = " ".join(commands)
+                response = await run_rcon(server, command)
+
+                #for status
+                if command == "status":
+
+                    status_response = await get_status(response)
+                    await ctx.send(status_response)
+
+                else:
+                #for everything else
+                    if response == "":
+                        await ctx.send("Command sent.")
+                    else:
+                        await ctx.send("```\n" + response + "\n```")
+            #eumvm
+            elif server == "eumvm":
+                command = " ".join(commands)
+                response = await run_rcon(server, command)
+
+                #for status
+                if command == "status":
+                    await ctx.send("```\n" + response.split('#', 1)[1] + "\n```")
+                else:
+                #for everything else
+                    if response == "":
+                        await ctx.send("Command sent.")
+                    else:
+                        await ctx.send("```\n" + response + "\n```")
+            #usmvm
+            elif server == "usmvm":
+                command = " ".join(commands)
+                response = await run_rcon(server, command)
+
+                #for status
+                if command == "status":
+                    await ctx.send("```\n" + response.split('#', 1)[1] + "\n```")
+                else:
+                #for everything else
+                    if response == "":
+                        await ctx.send("Command sent.")
+                    else:
+                        await ctx.send("```\n" + response + "\n```")
+            #rare edge case
+            else:
+                await ctx.send("How did it reach this part?")
+        #wrong server
+        else:
+            await ctx.send("Invalid server. See `?servers`.")
+
+    #status test command TODO
+    @command()
+    @has_any_role('Staff', 'Server Mods', 'Senior Staff', 'Fub')
+    async def status(self, ctx, server):
+        await ctx.trigger_typing()
+        if server == "us":
+            await ctx.send("us imp")
```

Comment:
i dont understand what this is supposed to accomplish? It just responds with  the phrase "us imp"?

Reply:
old code I forgot to remove. It was for seeing how a2s worked and what information I could pull from it. I ended up using this information to figure out !nextmap which I want to implement that into the !servers command we have now to display what map is next.